dev-core

```diff
@@ -7,6 +7,9 @@ MESG_LOG_LEVEL=${MESG_LOG_LEVEL:-debug}
 MESG_LOG_FORCECOLORS=${MESG_LOG_FORCECOLORS:-false}
 VERSION=local
 LDFLAGS="-X 'github.com/mesg-foundation/core/version.Version=$VERSION'"
+LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceEndpoint=https://ropsten.infura.io/v3/6115ae2531f64c04a9a392cf500e5fbe'"
```

Comment:
We can set it directly in the mesg.yaml of the service, in that case everone can deploy and use it without dev-core script.

Reply:
The problem is in the file `config/services.go`. It always replace the env.
It could create the env map dynamically only if the variable contains something...
It could be improved later on. Let's merge it this way for now.